ipld/amt/benches/amt_benchmark.rs

```diff
@@ -30,13 +30,11 @@ impl Serialize for BenchData {
 }
 
 impl<'de> Deserialize<'de> for BenchData {
-    #[allow(clippy::let_unit_value)]
     fn deserialize<D>(deserializer: D) -> Result<Self, <D as Deserializer<'de>>::Error>
     where
         D: Deserializer<'de>,
     {
-        let _s: () = Deserialize::deserialize(deserializer)?;
-
```

Comment:
I specifically silenced the warning instead of fixing the code because this is a benchmark and I'm pretty sure the motivation for this was to avoid compiler optimizations. May not apply though?

Reply:
I think even with this assignment the compiler should be smart enough to optimize it away if it wants to. But I can try to run it locally to see if I see a difference.

Reply:
@raulk It's good to go. When I remove this line completely, the benchmark will panic as there will be trailing data that didn't get deserialized => it's not optimized away.